NumExpr defaulting to 8 threads.
Importing GBM requirements failed. Not loading GBM model type. If you want to use GBM, install Ludwig's 'tree' extra.
Importing GBM requirements failed. Not loading LightGBM trainer. If you want to use LightGBM, install Ludwig's 'tree' extra.
ImportError (ray.py) from horovod.ray import RayExecutor failed with error: 
	No module named 'horovod'
Connecting to existing Ray cluster at address: 127.0.0.1:6379
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/1YVgUHXZeb'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/oyGrjIXt5a'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/5_lq9aXt5a'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/S8dr4RQxul'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/302Oj17IEC'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/HrWrjIXt5a'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/8I1r4RXXnK'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Foxt0Go8Lb'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/TfiBcmzkzH'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/a4v9tYsGvO'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/RO12'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/AERks4l'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/2Knk1KF'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/PomAbFU'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/SEuy'
While assessing potential image in is_image() for column Var194, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/lvza'
While assessing potential image in is_image() for column Var194, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/taul'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/LfvqpCtLOY'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/1K8T'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/h3JW'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/vsvk'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/L80O'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/AHgj'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/aD1X'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/lK27'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/uErj'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/G6s_'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/487l'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Jn8nx5A'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/A2frWlZ'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/qbzg2A0'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/DW3heVi'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/kJ1P82A'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/TeB8GsB'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/lUp7x5J'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/R9_sVcA'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/9GJGgoz'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/31DqtNX'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/y2LIM01bE1'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/msIqHk8toE'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/lL9c3pxXdL'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/itSGh3t'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/AnFl2Mwxzd'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/e1XhvTunuD'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/hOpRIhsUSP'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/NW71gM15FR'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/_jTP8ioIlJ'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/6xNyD9B'
While assessing potential image in is_image() for column Var200, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/n6_7gY_'
While assessing potential image in is_image() for column Var200, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/VDvP6WB'
While assessing potential image in is_image() for column Var200, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/oiSxHGd'
While assessing potential image in is_image() for column Var200, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/_VHQDJs'
While assessing potential image in is_image() for column Var200, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/qXbFKee'
While assessing potential image in is_image() for column Var200, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/smXZ'
While assessing potential image in is_image() for column Var201, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var201, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/1fxs'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/qnjZ'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/x3o3'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/RTc9'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/_YSY'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/sZLd'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/mTop'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/ydov'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/3zns'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/KwKi'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/9_Y1'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/STGZ'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/nr0d'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Y9Bl'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/z5Ry'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/zfpA'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/xQ2A'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Albh'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/C7DI'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/4nnx'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/YULl'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/sJzTlal'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/VpdQ'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/09_Q'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/zm5i'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/IYzP'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/haYg'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/sYC_'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/lVqb'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/me75fM6ugJ'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Kxdu'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/kIsH'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/uKAI'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Mtgm'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/L84s'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/NhsEn4L'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/FMSzZ91zL2'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/XfqtO3UdzaXh_'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/kmYOLQH'
While assessing potential image in is_image() for column Var214, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/ECxAcY5'
While assessing potential image in is_image() for column Var214, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/ewWoNO2'
While assessing potential image in is_image() for column Var214, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/6iFQbPB'
While assessing potential image in is_image() for column Var214, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/VajIlmE'
While assessing potential image in is_image() for column Var214, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/LHq13CE'
While assessing potential image in is_image() for column Var214, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/eGzu'
While assessing potential image in is_image() for column Var215, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/NGZxnJM'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/mAjdzWk'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/XTbPUYD'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/mAj3FSW'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/pMWcmoX'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/beK4AFX'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/mAjbk_S'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/kq0n8Bj'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/NGZokw4'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/kZJtVhC'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/FoMD'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/604y'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/BXUS'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/nJUo'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/y3rz'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/aOPp'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/9prz'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/otA4'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/5a5U'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/cJvF'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/UYBR'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/FzaX'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/b0vNgdR'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/dRiULkI'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Dndv4zN'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/THMQrGj'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Jptfql_'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/NvcKsQe'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/qxvYjZW'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/2LwVoxc'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Tvpip6Z'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/dMzYH7D'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/oslk'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/zCkv'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/yOuy5oW'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/tdtXJAQ'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/6ebzrB7'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/cObaF5M'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/2wCqFbB'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/T2oP3Og'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/ddLRDbC'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/bd1gE7h'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/hHJsvbM'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/_wyAgOW'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/LM8l689qOp'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/jySVZNlOJy'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/kG3k'
While assessing potential image in is_image() for column Var225, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/ELof'
While assessing potential image in is_image() for column Var225, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var225, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var225, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var225, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var225, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/453m'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/5Acm'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/uWr3'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/FSa2'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/7aLG'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Xa3G'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Qu4f'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Qcbd'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/RAYp'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/6fzt'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/F2FyR07IdsN7I'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/F2FcTt7IdMT_v'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/TCU50_Yjmm6GIBZ0lL_'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/iyHGyLCEkQ'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/SbOd7O8ky1wGNxp0Arj0Xs'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/am7c'
While assessing potential image in is_image() for column Var229, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var229, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var229, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/1YVgUHXZeb'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/oyGrjIXt5a'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/5_lq9aXt5a'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/S8dr4RQxul'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/302Oj17IEC'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/HrWrjIXt5a'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/8I1r4RXXnK'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Foxt0Go8Lb'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/TfiBcmzkzH'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/a4v9tYsGvO'
While assessing potential image in is_image() for column Var192, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/RO12'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/AERks4l'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/2Knk1KF'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/PomAbFU'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var193, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/SEuy'
While assessing potential image in is_image() for column Var194, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/lvza'
While assessing potential image in is_image() for column Var194, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/taul'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/LfvqpCtLOY'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var195, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/1K8T'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var196, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/h3JW'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/vsvk'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/L80O'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/AHgj'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/aD1X'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/lK27'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/uErj'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/G6s_'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/487l'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var197, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Jn8nx5A'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/A2frWlZ'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/qbzg2A0'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/DW3heVi'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/kJ1P82A'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/TeB8GsB'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/lUp7x5J'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/R9_sVcA'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/9GJGgoz'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/31DqtNX'
While assessing potential image in is_image() for column Var198, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/y2LIM01bE1'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/msIqHk8toE'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/lL9c3pxXdL'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/itSGh3t'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/AnFl2Mwxzd'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/e1XhvTunuD'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/hOpRIhsUSP'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/NW71gM15FR'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/_jTP8ioIlJ'
While assessing potential image in is_image() for column Var199, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/6xNyD9B'
While assessing potential image in is_image() for column Var200, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/n6_7gY_'
While assessing potential image in is_image() for column Var200, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/VDvP6WB'
While assessing potential image in is_image() for column Var200, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/oiSxHGd'
While assessing potential image in is_image() for column Var200, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/_VHQDJs'
While assessing potential image in is_image() for column Var200, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/qXbFKee'
While assessing potential image in is_image() for column Var200, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/smXZ'
While assessing potential image in is_image() for column Var201, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var201, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/1fxs'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/qnjZ'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/x3o3'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/RTc9'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/_YSY'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/sZLd'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/mTop'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/ydov'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/3zns'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/KwKi'
While assessing potential image in is_image() for column Var202, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/9_Y1'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var203, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/STGZ'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/nr0d'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Y9Bl'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/z5Ry'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/zfpA'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/xQ2A'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Albh'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/C7DI'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/4nnx'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/YULl'
While assessing potential image in is_image() for column Var204, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/sJzTlal'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/VpdQ'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/09_Q'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var205, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/zm5i'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/IYzP'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/haYg'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/sYC_'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/lVqb'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var206, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/me75fM6ugJ'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Kxdu'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var207, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/kIsH'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var208, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/uKAI'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var210, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Mtgm'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/L84s'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var211, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/NhsEn4L'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/FMSzZ91zL2'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/XfqtO3UdzaXh_'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var212, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/kmYOLQH'
While assessing potential image in is_image() for column Var214, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/ECxAcY5'
While assessing potential image in is_image() for column Var214, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/ewWoNO2'
While assessing potential image in is_image() for column Var214, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/6iFQbPB'
While assessing potential image in is_image() for column Var214, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/VajIlmE'
While assessing potential image in is_image() for column Var214, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/LHq13CE'
While assessing potential image in is_image() for column Var214, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/eGzu'
While assessing potential image in is_image() for column Var215, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/NGZxnJM'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/mAjdzWk'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/XTbPUYD'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/mAj3FSW'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/pMWcmoX'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/beK4AFX'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/mAjbk_S'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/kq0n8Bj'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/NGZokw4'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/kZJtVhC'
While assessing potential image in is_image() for column Var216, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/FoMD'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/604y'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/BXUS'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/nJUo'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/y3rz'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/aOPp'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/9prz'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/otA4'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/5a5U'
While assessing potential image in is_image() for column Var217, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/cJvF'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/UYBR'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var218, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/FzaX'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var219, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/b0vNgdR'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/dRiULkI'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Dndv4zN'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/THMQrGj'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Jptfql_'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/NvcKsQe'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/qxvYjZW'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/2LwVoxc'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Tvpip6Z'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/dMzYH7D'
While assessing potential image in is_image() for column Var220, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/oslk'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/zCkv'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var221, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/yOuy5oW'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/tdtXJAQ'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/6ebzrB7'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/cObaF5M'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/2wCqFbB'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/T2oP3Og'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/ddLRDbC'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/bd1gE7h'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/hHJsvbM'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/_wyAgOW'
While assessing potential image in is_image() for column Var222, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/LM8l689qOp'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/jySVZNlOJy'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var223, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/kG3k'
While assessing potential image in is_image() for column Var225, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/ELof'
While assessing potential image in is_image() for column Var225, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var225, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var225, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var225, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var225, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/453m'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/5Acm'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/uWr3'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/FSa2'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/7aLG'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Xa3G'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Qu4f'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/Qcbd'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var226, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/RAYp'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/6fzt'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var227, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/F2FyR07IdsN7I'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/F2FcTt7IdMT_v'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/TCU50_Yjmm6GIBZ0lL_'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/iyHGyLCEkQ'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/SbOd7O8ky1wGNxp0Arj0Xs'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var228, encountered exception: 'NoneType' object has no attribute 'tell'
[Errno 2] No such file or directory: '/Users/anne/ludwig/experiments/automl/validation/kdd/am7c'
While assessing potential image in is_image() for column Var229, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var229, encountered exception: 'NoneType' object has no attribute 'tell'
While assessing potential image in is_image() for column Var229, encountered exception: 'NoneType' object has no attribute 'tell'
Imbalance in target: minority/majority=0.01812258195886785
{'combiner': {'type': 'tabnet'},
 'hyperopt': {'executor': {'cpu_resources_per_trial': 1,
                           'num_samples': 10,
                           'scheduler': {'grace_period': 72,
                                         'max_t': 7200,
                                         'reduction_factor': 5,
                                         'time_attr': 'time_total_s',
                                         'type': 'async_hyperband'},
                           'time_budget_s': 7200,
                           'type': 'ray'},
              'goal': 'maximize',
              'metric': 'roc_auc',
              'output_feature': 'target',
              'parameters': {'combiner.bn_momentum': {'categories': [0.4,
                                                                     0.3,
                                                                     0.2,
                                                                     0.1,
                                                                     0.05,
                                                                     0.02],
                                                      'space': 'choice'},
                             'combiner.bn_virtual_bs': {'categories': [256,
                                                                       512,
                                                                       1024,
                                                                       2048,
                                                                       4096],
                                                        'space': 'choice'},
                             'combiner.num_steps': {'categories': [3,
                                                                   4,
                                                                   5,
                                                                   6,
                                                                   7,
                                                                   8,
                                                                   9,
                                                                   10],
                                                    'space': 'choice'},
                             'combiner.output_size': {'categories': [8,
                                                                     16,
                                                                     24,
                                                                     32,
                                                                     64,
                                                                     128],
                                                      'space': 'choice'},
                             'combiner.relaxation_factor': {'categories': [1.0,
                                                                           1.2,
                                                                           1.5,
                                                                           2.0],
                                                            'space': 'choice'},
                             'combiner.size': {'categories': [8,
                                                              16,
                                                              24,
                                                              32,
                                                              64],
                                               'space': 'choice'},
                             'combiner.sparsity': {'categories': [0.0,
                                                                  1e-06,
                                                                  0.0001,
                                                                  0.001,
                                                                  0.01,
                                                                  0.1],
                                                   'space': 'choice'},
                             'trainer.decay_rate': {'categories': [0.8,
                                                                   0.9,
                                                                   0.95],
                                                    'space': 'choice'},
                             'trainer.decay_steps': {'categories': [500,
                                                                    2000,
                                                                    8000,
                                                                    10000,
                                                                    20000],
                                                     'space': 'choice'},
                             'trainer.learning_rate': {'categories': [0.005,
                                                                      0.01,
                                                                      0.02,
                                                                      0.025],
                                                       'space': 'choice'}},
              'search_alg': {'random_state_seed': 42, 'type': 'hyperopt'}},
 'input_features': [{'column': 'Var1', 'name': 'Var1', 'type': 'number'},
                    {'column': 'Var2', 'name': 'Var2', 'type': 'number'},
                    {'column': 'Var3', 'name': 'Var3', 'type': 'number'},
                    {'column': 'Var4', 'name': 'Var4', 'type': 'number'},
                    {'column': 'Var5', 'name': 'Var5', 'type': 'number'},
                    {'column': 'Var6', 'name': 'Var6', 'type': 'number'},
                    {'column': 'Var7', 'name': 'Var7', 'type': 'number'},
                    {'column': 'Var9', 'name': 'Var9', 'type': 'number'},
                    {'column': 'Var10', 'name': 'Var10', 'type': 'number'},
                    {'column': 'Var11', 'name': 'Var11', 'type': 'number'},
                    {'column': 'Var12', 'name': 'Var12', 'type': 'number'},
                    {'column': 'Var13', 'name': 'Var13', 'type': 'number'},
                    {'column': 'Var14', 'name': 'Var14', 'type': 'number'},
                    {'column': 'Var16', 'name': 'Var16', 'type': 'number'},
                    {'column': 'Var17', 'name': 'Var17', 'type': 'number'},
                    {'column': 'Var18', 'name': 'Var18', 'type': 'number'},
                    {'column': 'Var19', 'name': 'Var19', 'type': 'number'},
                    {'column': 'Var21', 'name': 'Var21', 'type': 'number'},
                    {'column': 'Var22', 'name': 'Var22', 'type': 'number'},
                    {'column': 'Var23', 'name': 'Var23', 'type': 'number'},
                    {'column': 'Var24', 'name': 'Var24', 'type': 'number'},
                    {'column': 'Var25', 'name': 'Var25', 'type': 'number'},
                    {'column': 'Var26', 'name': 'Var26', 'type': 'number'},
                    {'column': 'Var27', 'name': 'Var27', 'type': 'number'},
                    {'column': 'Var28', 'name': 'Var28', 'type': 'number'},
                    {'column': 'Var29', 'name': 'Var29', 'type': 'number'},
                    {'column': 'Var30', 'name': 'Var30', 'type': 'number'},
                    {'column': 'Var33', 'name': 'Var33', 'type': 'number'},
                    {'column': 'Var34', 'name': 'Var34', 'type': 'number'},
                    {'column': 'Var35', 'name': 'Var35', 'type': 'number'},
                    {'column': 'Var36', 'name': 'Var36', 'type': 'number'},
                    {'column': 'Var37', 'name': 'Var37', 'type': 'number'},
                    {'column': 'Var38', 'name': 'Var38', 'type': 'number'},
                    {'column': 'Var40', 'name': 'Var40', 'type': 'number'},
                    {'column': 'Var41', 'name': 'Var41', 'type': 'number'},
                    {'column': 'Var43', 'name': 'Var43', 'type': 'number'},
                    {'column': 'Var44', 'name': 'Var44', 'type': 'number'},
                    {'column': 'Var45', 'name': 'Var45', 'type': 'number'},
                    {'column': 'Var46', 'name': 'Var46', 'type': 'number'},
                    {'column': 'Var47', 'name': 'Var47', 'type': 'number'},
                    {'column': 'Var49', 'name': 'Var49', 'type': 'number'},
                    {'column': 'Var50', 'name': 'Var50', 'type': 'number'},
                    {'column': 'Var51', 'name': 'Var51', 'type': 'number'},
                    {'column': 'Var53', 'name': 'Var53', 'type': 'number'},
                    {'column': 'Var54', 'name': 'Var54', 'type': 'number'},
                    {'column': 'Var56', 'name': 'Var56', 'type': 'number'},
                    {'column': 'Var57', 'name': 'Var57', 'type': 'number'},
                    {'column': 'Var58', 'name': 'Var58', 'type': 'number'},
                    {'column': 'Var59', 'name': 'Var59', 'type': 'number'},
                    {'column': 'Var60', 'name': 'Var60', 'type': 'number'},
                    {'column': 'Var61', 'name': 'Var61', 'type': 'number'},
                    {'column': 'Var62', 'name': 'Var62', 'type': 'number'},
                    {'column': 'Var63', 'name': 'Var63', 'type': 'number'},
                    {'column': 'Var64', 'name': 'Var64', 'type': 'number'},
                    {'column': 'Var65', 'name': 'Var65', 'type': 'number'},
                    {'column': 'Var66', 'name': 'Var66', 'type': 'number'},
                    {'column': 'Var67', 'name': 'Var67', 'type': 'number'},
                    {'column': 'Var68', 'name': 'Var68', 'type': 'number'},
                    {'column': 'Var69', 'name': 'Var69', 'type': 'number'},
                    {'column': 'Var70', 'name': 'Var70', 'type': 'number'},
                    {'column': 'Var71', 'name': 'Var71', 'type': 'number'},
                    {'column': 'Var72', 'name': 'Var72', 'type': 'number'},
                    {'column': 'Var73', 'name': 'Var73', 'type': 'number'},
                    {'column': 'Var74', 'name': 'Var74', 'type': 'number'},
                    {'column': 'Var75', 'name': 'Var75', 'type': 'number'},
                    {'column': 'Var76', 'name': 'Var76', 'type': 'number'},
                    {'column': 'Var77', 'name': 'Var77', 'type': 'number'},
                    {'column': 'Var78', 'name': 'Var78', 'type': 'number'},
                    {'column': 'Var80', 'name': 'Var80', 'type': 'number'},
                    {'column': 'Var81', 'name': 'Var81', 'type': 'number'},
                    {'column': 'Var82', 'name': 'Var82', 'type': 'number'},
                    {'column': 'Var83', 'name': 'Var83', 'type': 'number'},
                    {'column': 'Var84', 'name': 'Var84', 'type': 'number'},
                    {'column': 'Var85', 'name': 'Var85', 'type': 'number'},
                    {'column': 'Var86', 'name': 'Var86', 'type': 'number'},
                    {'column': 'Var87', 'name': 'Var87', 'type': 'number'},
                    {'column': 'Var88', 'name': 'Var88', 'type': 'number'},
                    {'column': 'Var89', 'name': 'Var89', 'type': 'number'},
                    {'column': 'Var90', 'name': 'Var90', 'type': 'number'},
                    {'column': 'Var91', 'name': 'Var91', 'type': 'number'},
                    {'column': 'Var92', 'name': 'Var92', 'type': 'number'},
                    {'column': 'Var93', 'name': 'Var93', 'type': 'number'},
                    {'column': 'Var94', 'name': 'Var94', 'type': 'number'},
                    {'column': 'Var95', 'name': 'Var95', 'type': 'number'},
                    {'column': 'Var96', 'name': 'Var96', 'type': 'number'},
                    {'column': 'Var97', 'name': 'Var97', 'type': 'number'},
                    {'column': 'Var98', 'name': 'Var98', 'type': 'number'},
                    {'column': 'Var99', 'name': 'Var99', 'type': 'number'},
                    {'column': 'Var100', 'name': 'Var100', 'type': 'number'},
                    {'column': 'Var101', 'name': 'Var101', 'type': 'number'},
                    {'column': 'Var102', 'name': 'Var102', 'type': 'number'},
                    {'column': 'Var103', 'name': 'Var103', 'type': 'number'},
                    {'column': 'Var104', 'name': 'Var104', 'type': 'number'},
                    {'column': 'Var105', 'name': 'Var105', 'type': 'number'},
                    {'column': 'Var106', 'name': 'Var106', 'type': 'number'},
                    {'column': 'Var107', 'name': 'Var107', 'type': 'number'},
                    {'column': 'Var108', 'name': 'Var108', 'type': 'number'},
                    {'column': 'Var109', 'name': 'Var109', 'type': 'number'},
                    {'column': 'Var110', 'name': 'Var110', 'type': 'number'},
                    {'column': 'Var111', 'name': 'Var111', 'type': 'number'},
                    {'column': 'Var112', 'name': 'Var112', 'type': 'number'},
                    {'column': 'Var113', 'name': 'Var113', 'type': 'number'},
                    {'column': 'Var114', 'name': 'Var114', 'type': 'number'},
                    {'column': 'Var115', 'name': 'Var115', 'type': 'number'},
                    {'column': 'Var116', 'name': 'Var116', 'type': 'number'},
                    {'column': 'Var117', 'name': 'Var117', 'type': 'number'},
                    {'column': 'Var118', 'name': 'Var118', 'type': 'category'},
                    {'column': 'Var119', 'name': 'Var119', 'type': 'number'},
                    {'column': 'Var120', 'name': 'Var120', 'type': 'number'},
                    {'column': 'Var121', 'name': 'Var121', 'type': 'number'},
                    {'column': 'Var122', 'name': 'Var122', 'type': 'number'},
                    {'column': 'Var123', 'name': 'Var123', 'type': 'number'},
                    {'column': 'Var124', 'name': 'Var124', 'type': 'number'},
                    {'column': 'Var125', 'name': 'Var125', 'type': 'number'},
                    {'column': 'Var126', 'name': 'Var126', 'type': 'number'},
                    {'column': 'Var127', 'name': 'Var127', 'type': 'number'},
                    {'column': 'Var128', 'name': 'Var128', 'type': 'number'},
                    {'column': 'Var129', 'name': 'Var129', 'type': 'number'},
                    {'column': 'Var130', 'name': 'Var130', 'type': 'number'},
                    {'column': 'Var131', 'name': 'Var131', 'type': 'number'},
                    {'column': 'Var132', 'name': 'Var132', 'type': 'number'},
                    {'column': 'Var133', 'name': 'Var133', 'type': 'number'},
                    {'column': 'Var134', 'name': 'Var134', 'type': 'number'},
                    {'column': 'Var135', 'name': 'Var135', 'type': 'number'},
                    {'column': 'Var136', 'name': 'Var136', 'type': 'number'},
                    {'column': 'Var137', 'name': 'Var137', 'type': 'number'},
                    {'column': 'Var138', 'name': 'Var138', 'type': 'number'},
                    {'column': 'Var139', 'name': 'Var139', 'type': 'number'},
                    {'column': 'Var140', 'name': 'Var140', 'type': 'number'},
                    {'column': 'Var142', 'name': 'Var142', 'type': 'number'},
                    {'column': 'Var143', 'name': 'Var143', 'type': 'number'},
                    {'column': 'Var144', 'name': 'Var144', 'type': 'number'},
                    {'column': 'Var145', 'name': 'Var145', 'type': 'number'},
                    {'column': 'Var146', 'name': 'Var146', 'type': 'number'},
                    {'column': 'Var147', 'name': 'Var147', 'type': 'number'},
                    {'column': 'Var148', 'name': 'Var148', 'type': 'number'},
                    {'column': 'Var149', 'name': 'Var149', 'type': 'number'},
                    {'column': 'Var150', 'name': 'Var150', 'type': 'number'},
                    {'column': 'Var151', 'name': 'Var151', 'type': 'number'},
                    {'column': 'Var152', 'name': 'Var152', 'type': 'number'},
                    {'column': 'Var153', 'name': 'Var153', 'type': 'number'},
                    {'column': 'Var154', 'name': 'Var154', 'type': 'number'},
                    {'column': 'Var155', 'name': 'Var155', 'type': 'number'},
                    {'column': 'Var156', 'name': 'Var156', 'type': 'number'},
                    {'column': 'Var157', 'name': 'Var157', 'type': 'number'},
                    {'column': 'Var158', 'name': 'Var158', 'type': 'number'},
                    {'column': 'Var159', 'name': 'Var159', 'type': 'number'},
                    {'column': 'Var160', 'name': 'Var160', 'type': 'number'},
                    {'column': 'Var161', 'name': 'Var161', 'type': 'number'},
                    {'column': 'Var162', 'name': 'Var162', 'type': 'number'},
                    {'column': 'Var163', 'name': 'Var163', 'type': 'number'},
                    {'column': 'Var164', 'name': 'Var164', 'type': 'number'},
                    {'column': 'Var165', 'name': 'Var165', 'type': 'number'},
                    {'column': 'Var166', 'name': 'Var166', 'type': 'number'},
                    {'column': 'Var168', 'name': 'Var168', 'type': 'number'},
                    {'column': 'Var170', 'name': 'Var170', 'type': 'number'},
                    {'column': 'Var171', 'name': 'Var171', 'type': 'number'},
                    {'column': 'Var172', 'name': 'Var172', 'type': 'number'},
                    {'column': 'Var173', 'name': 'Var173', 'type': 'number'},
                    {'column': 'Var174', 'name': 'Var174', 'type': 'number'},
                    {'column': 'Var176', 'name': 'Var176', 'type': 'number'},
                    {'column': 'Var177', 'name': 'Var177', 'type': 'number'},
                    {'column': 'Var178', 'name': 'Var178', 'type': 'number'},
                    {'column': 'Var179', 'name': 'Var179', 'type': 'number'},
                    {'column': 'Var180', 'name': 'Var180', 'type': 'number'},
                    {'column': 'Var181', 'name': 'Var181', 'type': 'number'},
                    {'column': 'Var182', 'name': 'Var182', 'type': 'number'},
                    {'column': 'Var183', 'name': 'Var183', 'type': 'number'},
                    {'column': 'Var184', 'name': 'Var184', 'type': 'number'},
                    {'column': 'Var186', 'name': 'Var186', 'type': 'number'},
                    {'column': 'Var187', 'name': 'Var187', 'type': 'number'},
                    {'column': 'Var188', 'name': 'Var188', 'type': 'number'},
                    {'column': 'Var189', 'name': 'Var189', 'type': 'number'},
                    {'column': 'Var190', 'name': 'Var190', 'type': 'number'},
                    {'column': 'Var191', 'name': 'Var191', 'type': 'category'},
                    {'column': 'Var192', 'name': 'Var192', 'type': 'category'},
                    {'column': 'Var193', 'name': 'Var193', 'type': 'category'},
                    {'column': 'Var194', 'name': 'Var194', 'type': 'category'},
                    {'column': 'Var195', 'name': 'Var195', 'type': 'category'},
                    {'column': 'Var196', 'name': 'Var196', 'type': 'category'},
                    {'column': 'Var197', 'name': 'Var197', 'type': 'category'},
                    {'column': 'Var198', 'name': 'Var198', 'type': 'category'},
                    {'column': 'Var199', 'name': 'Var199', 'type': 'category'},
                    {'column': 'Var200', 'name': 'Var200', 'type': 'category'},
                    {'column': 'Var201', 'name': 'Var201', 'type': 'category'},
                    {'column': 'Var202', 'name': 'Var202', 'type': 'category'},
                    {'column': 'Var203', 'name': 'Var203', 'type': 'category'},
                    {'column': 'Var204', 'name': 'Var204', 'type': 'category'},
                    {'column': 'Var205', 'name': 'Var205', 'type': 'category'},
                    {'column': 'Var206', 'name': 'Var206', 'type': 'category'},
                    {'column': 'Var207', 'name': 'Var207', 'type': 'category'},
                    {'column': 'Var208', 'name': 'Var208', 'type': 'category'},
                    {'column': 'Var210', 'name': 'Var210', 'type': 'category'},
                    {'column': 'Var211', 'name': 'Var211', 'type': 'category'},
                    {'column': 'Var212', 'name': 'Var212', 'type': 'category'},
                    {'column': 'Var213', 'name': 'Var213', 'type': 'category'},
                    {'column': 'Var214', 'name': 'Var214', 'type': 'category'},
                    {'column': 'Var215', 'name': 'Var215', 'type': 'category'},
                    {'column': 'Var216', 'name': 'Var216', 'type': 'category'},
                    {'column': 'Var217', 'name': 'Var217', 'type': 'category'},
                    {'column': 'Var218', 'name': 'Var218', 'type': 'category'},
                    {'column': 'Var219', 'name': 'Var219', 'type': 'category'},
                    {'column': 'Var220', 'name': 'Var220', 'type': 'category'},
                    {'column': 'Var221', 'name': 'Var221', 'type': 'category'},
                    {'column': 'Var222', 'name': 'Var222', 'type': 'category'},
                    {'column': 'Var223', 'name': 'Var223', 'type': 'category'},
                    {'column': 'Var224', 'name': 'Var224', 'type': 'category'},
                    {'column': 'Var225', 'name': 'Var225', 'type': 'category'},
                    {'column': 'Var226', 'name': 'Var226', 'type': 'category'},
                    {'column': 'Var227', 'name': 'Var227', 'type': 'category'},
                    {'column': 'Var228', 'name': 'Var228', 'type': 'category'},
                    {'column': 'Var229', 'name': 'Var229', 'type': 'category'}],
 'output_features': [{'column': 'target', 'name': 'target', 'type': 'binary'}],
 'preprocessing': {'split': {'column': 'split', 'type': 'fixed'}},
 'trainer': {'batch_size': 'auto',
             'decay': True,
             'decay_rate': 0.8,
             'decay_steps': 20000,
             'learning_rate': 'auto',
             'learning_rate_scaling': 'sqrt',
             'optimizer': {'type': 'adam'},
             'validation_field': 'target',
             'validation_metric': 'roc_auc'}}
